docs/Configuring_openshift_virtualization_vm_for_dpdk.md

```diff
@@ -0,0 +1,281 @@
+
+## Prerequisites
+
+- [Deploy](https://www.redhat.com/en/technologies/cloud-computing/openshift/deploy-red-hat-openshift) an OpenShift Container Platform cluster (OCP v4.13)
+- [Deploy](https://docs.openshift.com/container-platform/4.12/virt/install/installing-virt-cli.html) OpenShift Virtualization plugin.tain
+- [Create](https://docs.openshift.com/container-platform/4.8/applications/projects/working-with-projects.html) a project on the cluster where the VM will run.
+- [Deploy](https://docs.openshift.com/container-platform/4.12/networking/hardware_networks/installing-sriov-operator.html) SR-IOV Network Operator.
+- The relevant worker nodes must have:
+  - SR-IOV enabled NICs.
+  - Allocatable Hugepages.
+
+*Note*: In this article an [Intel X710](https://www.dell.com/en-us/shop/intel-x710-quad-port-10gb-direct-attach-sfp-converged-network-adapter-full-height/apd/540-bbiw/networking#tabs_section) Network Adapter was used.
+
+## Configuration
+
+The configuration chapter will consist of three parts:
+- Cluster configuration - requires cluster-admin permissions.
+- VM spec configuration - requires project-admin permissions.
+- Guest configuration - based on RHEL8 OS.
+
+### Cluster configurations
+
+This step consists of:
+- Deploying a `vfio-pci` typed [SriovNetworkNodePolicy](https://docs.openshift.com/container-platform/4.12/networking/hardware_networks/configuring-sriov-device.html) and [SriovNetwork](https://docs.openshift.com/container-platform/4.12/networking/hardware_networks/configuring-sriov-device.html#cnf-assigning-a-sriov-network-to-a-vrf_configuring-sriov-device) using the [sriov-network-operator](https://github.com/openshift/sriov-network-operator) that is deployed on the cluster.
+Doing so will result in the creation of a [network-attachment-definition](https://docs.openshift.com/container-platform/4.12/virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.html) that will be used by the OpenShift Virtualization VM in the next steps.
+- [Checking](https://docs.openshift.com/container-platform/4.12/post_installation_configuration/machine-configuration-tasks.html#checking-mco-status_post-install-machine-configuration-tasks) the node has a machineConfigPool, and [creating](https://access.redhat.com/solutions/5688941) one if needed.
+- Creating a [performance-profile](https://docs.openshift.com/container-platform/4.12/scalability_and_performance/cnf-create-performance-profiles.html) using the [Node-Tuning-Operator](https://docs.openshift.com/container-platform/4.12/scalability_and_performance/using-node-tuning-operator.html).
+Doing so will create a [runtimeClass](https://kubernetes.io/docs/concepts/containers/runtime-class/) that would be used by the OpenShift Virtualization VM in the next steps.
+
+In order to configure the cluster according to these steps please follow this OCP documentation.
```

Comment:
Missing a link that was present on the google doc:
```suggestion
In order to configure the cluster according to these steps please follow this [OCP documentation](https://docs.openshift.com/container-platform/4.12/networking/hardware_networks/using-dpdk-and-rdma.html).
```

Reply:
DONE